tech.json

```diff
@@ -54,13 +54,18 @@
             {
               "name": "canQuickLowTideWalljumpWaterEscape",
               "requires": [
-                 "canSuitlessMaridia",
-                 "canWalljump"
-                ],
+                "canSuitlessMaridia",
+                "canWalljump"
+              ],
               "note": "The ability to get out of water by walljumping while the water is at its lowest point, and to do so in a timely manner, e.g. before a Yapping Maw drags you into a sand pit.",
               "devNote":"This is so specific. Can we do something to remove it?"
             }
           ]
+        },
+        {
+          "name": "canPixelPreciseTrick",
```

Comment:
This could be confused with many other pixel perfect tricks. maybe canStandPixelPerfect?

Reply:
The more that I think about this, the more that I think the tech should be shooting through the wall. Standing is very vague and could be applied to many strats.

Reply:
It is mostly a stand-in name.  A general `pixelPreciseTrick` tech can be discussed elsewhere.